spec/no_reinstall.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e -x
+
+! grep "job: install bind" y2log && echo "AUTOYAST OK" "job: install bind" y2log && echo "AUTOYAST OK"
```

Comment:
I don't get this line. Is the last part ok (starting on the second `job: install bind`)?